requirements/base.txt

```diff
@@ -163,6 +168,8 @@ django-formtools==2.5.1
     # via django-two-factor-auth
 django-hijack==3.4.1
     # via -r requirements/base.in
+django-import-export==2.9.0
```

Comment:
why 2.9.0 and not 3.3.x?

Reply:
I guess I need to pass more dependencies to `-P`